scripts/code_size_compare.py

```diff
@@ -120,6 +121,40 @@ def _gen_code_size_csv(self, revision, git_worktree_path):
             data = line.split()
             csv_file.write("{}, {}\n".format(data[5], data[3]))
 
+    def _gen_code_size_csv_details(self, revision, git_worktree_path):
+        """Generate symbol-level code size csv file"""
+
+        csv_fname = revision + "_symbols.csv"
+        if revision == "current":
+            print("Measuring code size in current work directory.")
+        else:
+            print("Measuring code size for", revision)
+        result = subprocess.check_output(
+            ["nm library/*.a --size-sort"], cwd=git_worktree_path, shell=True
+        )
+        size_text = result.decode()
```

Comment:
This could be simplified:
```suggestion
        size_text= subprocess.check_output(
            ["nm library/*.a --size-sort"], cwd=git_worktree_path, shell=True, universal_newlines=True
        )
```